### PR TITLE
checksum fail notice modification

### PR DESCRIPTION
**Pull request by mrxkon**
About #56 .

Adding an extra line, explaining about some of the common false-positive files.

### CODE REVIEW

**Comment by Clorith** [comment]
Let's change that up slightly, I think.

> Some files may have been modified by your translation, try doing a __Reinstall__ (don't worry, it only replaces WordPress' own files) and test again.

I noticed if I manually click the reinstall button in the update screen, it grabs the files as expected based on my locale, and the checksums pass as they should. (Some input from @tobifjellner who appears to know a bit more about it than I would be more than welcome here to confirm that I'm not just having some weird behavior because I'm me).

File: includes/class-health-check-files-integrity.php [hunk 1]
```diff
@@ -110,6 +110,7 @@ static function create_the_response( $files ) {
 		} else {
 			$output .= '<div class="notice notice-error inline"><p>';
 			$output .= __( 'It appears that some files may have been modified.', 'health-check' );
+			$output .= '<br/>' . __( 'Files such as versions.php, wp-config-sample.php, license.txt, readme.html, might have been modified by translations.', 'health-check' );
```
Comment (bueltge) [inline includes/class-health-check-files-integrity.php]: `esc_html__` is necessary. Also `<br/>` is xhtml, `<br>` is a better way.

**Comment by mrxkon** [comment]
Thank you @bueltge I'll update your notices on the next commit.

@Clorith if we view the whole notice now it would look like this:

```
It appears that some files may have been modified.
Some files may have been modified by your translation, try doing a Re-install (don't worry, it only replaces WordPress' own files) and test again.
```

The use of `some files may have been modified` twice just throws me off a bit even as a non-English person. Should we merge them into 1 somehow or just remove the original `It appears that some files may have been modified.` completely ?

**Comment by tobifjellner** [comment]
Suggestion:
It appears as if some files may have been modified.
One possible reason for this may be that your installation contains translated versions. An easy way to clear this is to reinstall WordPress. Don’t worry. This will only affect WordPress’ own files, not your themes, plugins or uploaded media.

**Comment by Clorith** [comment]
I like what Tor-Björn suggests, it makes sense to err on the side of more text to explain a users what is going on in a situation like this.

**Pull request merged**